### PR TITLE
Update fatal error designs

### PR DESCRIPTION
**Pull request by buberdds**
https://app.clickup.com/t/8693qqdp3

Screenshot from 2024-03-28 10-09-10
Screenshot from 2024-03-28 10-09-34
Screenshot from 2024-03-28 10-09-57

### CODE REVIEW

**Comment by buberdds** [comment]
Hey @donouwens, can you validate this `https://da444f64.oasis-wallet.pages.dev/` ?
Designs are using proprietary font, different iconography etc so I had to improvise in a little bit. Anyway mobile and desktop is waiting for your feedback and acceptance :salute:

**Comment by buberdds** [comment]
Now I am aware ;) I assume, you are using Safari ?

File: src/app/components/FatalErrorHandler/index.tsx [hunk 1]
```diff
@@ -3,29 +3,52 @@
...
 import { useSelector } from 'react-redux'
-
+import { normalizeColor } from 'grommet/es6/utils'
 import { ResponsiveLayer } from '../ResponsiveLayer'
+import logotype from '../../../../public/logo192.png'
+
+const StyledTextArea = styled(TextArea)`
+  // Opacity is a workaround for browsers anti-aliasing issue triggered by border-radius.
+  // Without it text is blurry.
+  background-color: ${({ theme }) => `${normalizeColor('brand-background-light', theme)}fe`};
```
Comment (lukaw3d) [inline src/app/components/FatalErrorHandler/index.tsx:30]: That's so weird!
It doesn't happen in Firefox, only Chrome. Did you find any issues about it?

**Review by lukaw3d — APPROVED** [review]

File: src/app/components/FatalErrorHandler/index.tsx (hunk 1 above)
Comment (buberdds) [reply src/app/components/FatalErrorHandler/index.tsx:30]: I did not look at any Chromium issue tracker yet.

**Pull request merged**